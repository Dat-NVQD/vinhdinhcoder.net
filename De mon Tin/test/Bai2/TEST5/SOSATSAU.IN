500
25070830270709479238561397108022623017038721639219
72492299417033948887919097283561213564104499834858
84346388749459794475654968841642676777533575282634
52705490355122113735103905451667596563558348644607
24994098602232714268764517997733317129086572599028
13747728596304393980495692693097443394153771852146
22010052119963000389465790624439010527817798368718
85296638488644574081023744748750831539132892808466
87972198839173430199031133365488747162723742397542
81583159632154435811340594114640716524085507083739
