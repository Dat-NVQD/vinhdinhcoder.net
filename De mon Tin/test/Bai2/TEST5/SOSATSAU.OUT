25070830270709479238561397108022623017038721639219724922994170339488879190972835612135641044998348588434638874945979447565496884164267677753357528263452705490355122113735103905451667596563558348644607249940986022327142687645179977333171290865725990281374772859630439398049569269309744339415377185214622010052119963000389465790624439010527817798368718852966384886445740810237447487508315391328928084668797219883917343019903113336548874716272374239754281583159632154435811340594114640716524085507083793
